string                      agent_name
mac_ros_bridge/Position     pos
string                      well_type
bool                        built